docs/source/examples/spot-jobs.rst

```diff
@@ -176,7 +176,7 @@ We also set :code:`--run_name` to :code:`$SKYPILOT_TASK_ID` so that the logs for
 to the same run in Weights & Biases.
 
 .. note::
-  The environment variable :code:`$SKYPILOT_TASK_ID` (example: "sky-2022-10-06-05-17-09-750781_spot_id-22") can be used to identify the same job, i.e., it is kept identical across all
+  The environment variable :code:`$SKYPILOT_TASK_ID` (example: "sky-2022-10-06-05-17-09-750781_bert-qa_spot_id-22-1") can be used to identify the same job, i.e., it is kept identical across all
```

Comment:
nit: Name in YAML is `bert_qa`. Should it be that or `bert-qa`?

Reply:
Ahh, that is because we override the `-n` in the command line. I changed the `name:` to `bert-qa` in the task to reduce the confusion now : )